tests/test_vector3.cpp

```diff
@@ -0,0 +1,165 @@
+//
+// Created by ajer on 30.12.18.
+//
+
+#include "pch.h"
+#include "gtest/gtest.h"
+#include "math/vec3.h"
+
+
+TEST(Getter, Vec3_GetX) { //NOLINT
+    Bald::Math::Vec3 A(66.0f, 3.4f, 1.0f);
+
+    ASSERT_FLOAT_EQ(66.0f, A.GetX());
+}
+
+
+TEST(Getter, Vec3_GetY) { //NOLINT
+    Bald::Math::Vec3 A(2.0f, -23.33f, 2.0f);
+
+    ASSERT_FLOAT_EQ(-23.33f, A.GetY());
+}
+
+
+TEST(Getter, Vec3_GetZ) { //NOLINT
+    Bald::Math::Vec3 A(1.0f, -21.2f, -5.3f);
+
+    ASSERT_FLOAT_EQ(-5.3f, A.GetZ());
+}
+
+
+TEST(NormalizeVector, Vec3_StaticUnitVec) { //NOLINT
+    Bald::Math::Vec3 A = Bald::Math::Vec3::MakeUnitVec(Bald::Math::Vec3(2.0f, 3.0f, 6.0f));
+
+    float x = 2.0f / 7.0f; // 0.28571
+    float y = 3.0f / 7.0f; // 0.42857
+    float z = 6.0f / 7.0f; // 0.85714
+
+    ASSERT_EQ(Bald::Math::Vec3(x, y, z), A);
```

Comment:
Why not move equations into the Vec3 constructor?

Reply:
moved